app/models/reservation.rb

```diff
@@ -89,6 +89,15 @@ def self.not_this_reservation(reservation)
 
   scope :ongoing, -> { not_ended.where("actual_start_at <= ?", Time.current) }
 
+  scope :current_in_use, lambda {
```

Comment:
There's already an `active` scope, but that includes everything that has been purchased but restricts it to only user reservations (i.e. it excludes admin and offline reservations). I wouldn't mind a better name if someone has an idea.